admin/src/containers/Login/Auth.js

```diff
@@ -0,0 +1,45 @@
+import cookie from 'cookie';
+import config from '../../config';
```

Comment:
is this a container? It doesn't seem to render anything?

Reply:
It's more like a helper function. Is a dependency for Login container but could be used in other containers to get logged user information. Could be move to a `lib` directory, perhaps is more accurate.